docs/tutorials/annotation/index.md

```diff
@@ -1,3 +1,4 @@
+(annotation)=
 # Annotation
 
 This section contains tutorials showing how to annoate images and videos using the napari points, labels, or shapes layers.
```

Comment:
I know this isn't your typo but we should probably fix it now to say `annotate` not `annoate`